spec/requests/comment_spec.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Comments', type: :request do
+  describe 'NEW post comment' do
+    let!(:user) { create(:user) }
+    let!(:newpost) { create(:post) }
+    let(:content) { Faker::Lorem.paragraph }
+    let(:comment_params) do
+      { comment: { content:, commentable_type: newpost.class.name, commentable_id: newpost.id } }
+    end
+
+    before do
+      sign_in user
+    end
+
+    it 'creates a new Post Comment (through AJAX)' do
+      expect do
+        post comments_path, params: comment_params, xhr: true
+      end.to change(Comment, :count).by(1)
+    end
+
+    it 'responds with JavaScript' do
+      post comments_path, params: comment_params, xhr: true
+      expect(response.content_type).to eq('text/javascript; charset=utf-8')
+      expect(response.body).to include(content)
+    end
+
+    it 'creates a new Post Comment (through HTML)' do
+      expect do
+        post comments_path, params: comment_params
+      end.to change(Comment, :count).by(1)
+    end
+
+    it 'responds with HTML' do
+      post comments_path, params: comment_params
+      expect(response).to redirect_to(post_path(newpost.id))
+      follow_redirect!
+      expect(response.body).to include(content)
+    end
```

Comment:
Apart from these tests which are perfect, you should test that the created comment has the correct attributes, that is, it got associated with the correct user, the correct post and the correct content